relay-server/src/actors/processor.rs

```diff
@@ -3075,11 +3150,60 @@ impl EnvelopeProcessorService {
         }
     }
 
+    #[cfg(not(feature = "processing"))]
+    fn handle_encode_metric_meta(&self, message: EncodeMetricMeta) {
+        let EncodeMetricMeta { scoping, meta } = message;
+
+        let upstream = self.inner.config.upstream_descriptor();
+        let dsn = crate::extractors::PartialDsn {
```

Comment:
In a follow-up, let's move this to a method on `EnvelopeProcessorService` since this is duplicated three times.